src/utils/uuid.ts

```diff
@@ -0,0 +1,8 @@
+import { v5 as uuid5 } from 'uuid';
+
+// Generates UUIDv5, used to consistently generate the same UUID for
+// a specific identifier and namespace
+export function generateUuid5(identifier: any, namespace: any = ''): string {
```

Comment:
just a suggestion, maybe just `uuid5` would be sufficient? I'm OK with both :)

Reply:
and args can be `any`? or only `strings`?

Reply:
I have updated the param type for both `identifier` and `namespace` to `string | number`